tfx/components/util/tfxio_utils_test.py

```diff
@@ -306,7 +307,7 @@ def test_get_tf_dataset_factory_from_artifact(self):
     dataset_factory = tfxio_utils.get_tf_dataset_factory_from_artifact(
         [examples], _TELEMETRY_DESCRIPTORS)
     self.assertIsInstance(dataset_factory, Callable)
-    self.assertEqual(tf.data.Dataset,
+    self.assertEqual('tf.data.Dataset',
```

Comment:
this stringify of the signature is a result of adding: `from __future__ import annotations`